Sources/MapboxNavigation/UIImage.swift

```diff
@@ -43,4 +43,31 @@ extension UIImage {
 
         return scaledImage
     }
+
+    // Produce a copy of the image with tint color applied.
+    // Useful for deployment to iOS versions prior to 13 where tinting support was added to UIImage natively.
+    func tint(_ tintColor: UIColor) -> UIImage {
+        let imageSize = size
+        let imageScale = scale
+        let contextBounds = CGRect(origin: .zero, size: imageSize)
+
+        UIGraphicsBeginImageContextWithOptions(imageSize, false, imageScale)
```

Comment:
Can we use modern `UIGraphicsImageRenderer` API here?

Reply:
This method landed as is in #2873. #3893 tracks modernizing this method.